ckanext/benap/benap_schema.json

```diff
@@ -548,15 +548,15 @@
     },
     {
       "field_name": "theme",
+      "preset": "hidden",
       "label": {
         "en": "Dataset theme",
         "fr": "Thème du jeu de données",
         "nl": "Thema dataset",
         "de": "Thema des Datensatzes"
       },
-
+      "default": "http://publications.europa.eu/resource/authority/data-theme/TRAN",
       "display_property": "dcat:theme",
-      "preset": "select",
       "choices_helper": "benap_ontology_helper",
```

Comment:
Aangezien de waarde default wordt, mag de "choices_helper" worden weggelaten

Reply:
Nee gata hier niet, die 2 helpers worden gebruikt om het correct label en value weer te geven in de dataset tabel. Het is een speciaal veld waarbij de waarde wordt aangepast obv de taal van de pagina. Bijvoorbeeld 'Transport' in het Engels als thema en in het Nederlands bijvoorbeeld 'Verkeer'. Klinkt een beetje vreemd met de naamgeving 'choices' dus ik snap je vraag wel maar het kan niet weggelaten worden.

Reply:
Aanpassen lijkt me ook overbodig aangezien het voorheen ook al op deze manier geprogrameerd werd. Het veld blijft hetzelfde, enkel is het niet zichtbaar nu en is er een default waarde.